libs/payments/events/src/lib/emitter.types.ts

```diff
@@ -13,15 +13,14 @@ export type CheckoutPaymentEvents = CommonMetrics & {
   paymentProvider?: PaymentProvidersType;
 };
 
-export type SubscriptionEnded = {
-  productId: string;
+export type SubscriptionEndedEvents = {
```

Comment:
Why was `productId` removed? Seems like it's just being fetched again by the EmitterService?